src/core/SparseMatrix.cpp

```diff
@@ -329,6 +329,23 @@ T SparseMatrix<T>::Value( Int index ) const EL_NO_RELEASE_EXCEPT
     return vals_[index];
 }
 
+template< typename T>
+T SparseMatrix<T>::Get( Int row, Int col) const EL_NO_RELEASE_EXCEPT
+{
+    Int index = Offset( row, col);
+    if( Row(index) != row || Col(index) != col){
+            return T(0); 
+    }
+    return Value( Offset( row, col)); 
+}
+
+template< typename T>
+void SparseMatrix<T>::Set( Int row, Int col, T val) EL_NO_RELEASE_EXCEPT
```

Comment:
The current implementation of `Set` actually performs an `Update` (that is, if entry `(row,col)` was already nonzero, it adds `val` to it rather than assigning `val`).